app/src/main/java/org/cru/godtools/activity/AboutActivity.java

```diff
@@ -20,7 +20,7 @@
 public final class AboutActivity extends BasePlatformActivity {
     private static final String TAG_MAIN_FRAGMENT = "mainFragment";
 
-    public static void start(@NonNull final Context context) {
+    public static void start(@NonNull final Activity context) {
```

Comment:
small nit, just to be thorough I would rename the `context` param to `activity`

Reply:
OK